scipy/sparse/_sputils.py

```diff
@@ -258,6 +262,7 @@ def isshape(x, nonneg=False, *, allow_1d=False) -> bool:
     return True
 
 
+
```

Comment:
nit: 2 blank lines is enough, this is the 3rd one

Reply:
Removed at https://github.com/scipy/scipy/pull/21197/commits/89ffb7f50f2fb22ea3a33aa5e0c3f6ad29ca4bb3.